### ISSUE TITLE
Port remaining clarity a11y fixes to other themes

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5334]
Some of the fixes made during the a11y review of clarity did not port to the other themes, and they need to as they they were 1.9 fixes.

aka I opened up 1.9.x an found problems I remember fixing, then traced these fixes back to the clarity branch work.

**Comment by StephDriver** [comment]
1. d1530f25b — Add dyslexia CSS variables 
2. fee88b1b7 — Core dyslexia styles + reading options layout + sticky TOC restructure
3. 03e40087b — Add !important to dyslexia colors for specificity override
4. 727453a67 — Exclude Font Awesome icons from dyslexia font change

**Comment by StephDriver** [comment]
closed in favour of #5337 because that will be rewriting the dyslexia mode to create a more general text formatting UI. So these missing styles will be moot.
